chirps_utility.py

```diff
@@ -0,0 +1,83 @@
+from chirps import *
+from csv_utility import *
+from user_utility import *
+from random_id_generator import *
+
+class ChirpsUtility:
+
+	def view_chirps(chirps_dict, current_user):
+		"""
+		Prints out all public and private chirps that the current user is a part of
+
+		Arg- chirps dictionary (from CSV file), currently selected user
+		"""
+		counter = 1
+		if current_user != None:
+			print("")
+			print(" ~ PRIVATE CHIRPS ~ ")
```

Comment:
If you'd like a line break before a printed line, instead of printing an empty string before it, you could also stick a newline character `\n` at the start of the line you want the break before.